src/pages/LoginPage/index.js

```diff
@@ -30,10 +30,7 @@ const LoginPage = () => {
     }
   };
 
-  useEffect(() => {
-    if (supabaseClient.auth.user())
-      navigate("/listingspage");
-  });
```

Comment:
Should keep this code, or replace with code that redirects logged-in users out of the login/register pages.